files/sensu_slo.rb

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env ruby
+#
+# Sensu Handler: sensu_slo
+#
+# This handler calculate the age of a check by subtracting the execution
+# time of the check with the current time on the Sensu server. This can
+# be used to monitor check latency and also, by evaluating the interval
+# time, used to give and idea of the number of succesfull checks
+# navigating Sensu's message bus.
+
+require 'rubygems' if RUBY_VERSION < '1.9.0'
+require 'sensu-handler'
+
+$env_dir = "/nail/etc/"
+$default_dims_files = [
+  "runtimeenv",
+  "ecosystem",
+  "region",
+  "habitat"
+]
+
+class SensuSLOHandler < Sensu::Handler
+
+  def handle
+
+    metric_name = settings['sensu_slo']['metric_name'] || 'sensu.check_age"'
+
+    statsite_host = settings['sensu_slo']['statsite_host'] || '127.0.0.1'
+    statsite_port = settings['sensu_slo']['statsite_port'] || 8125
+
+    # Extract the name of the check and the client which ran it
+    if @event["check"] != nil
```

Comment:
Why not do if @event["check"].nill? here - you can just print and exit in that case, then you don't need the else branch at all (the stuff in it becomes tests at the top level with no nested ifs) and this would reduce cyclomatic complexity